src/modules/users/components/TokenActivation/TokenActivationPopover.tsx

```diff
@@ -4,9 +4,8 @@ import Popover, { PopoverChildFn } from '~core/Popover';
 
 import { removeValueUnits } from '~utils/css';
 
-import TokenActivationContent, {
-  TokensTabProps,
-} from './TokenActivationContent/index';
+import TokenActivationContent from './TokenActivationContent/index';
```

Comment:
This `.../index` can be removed so that it's just `./TokenActivationContent`

Reply:
I'll clean this up. thx